Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -109,11 +112,18 @@ struct ScheduleTransferSummaryData: ScheduleTransferSectionData {
             let grossTransferAmount = transferAmountFormattedDouble + feeAmountFormattedDouble
 
             rows.append((title: "transfer_amount_confirmation".localized(),
-                         value: String(grossTransferAmount).format(with: destinationCurrency)))
-            rows.append((title: "transfer_fee_confirmation".localized(),
-                         value: feeAmount.format(with: destinationCurrency)))
+                         value: String(format: "%@ %@",
+                                       String(grossTransferAmount)
+                            .format(withActualDecimal: destinationCurrency), destinationCurrency)))
+            if feeAmount != "0.00" {
```

Comment:
move this condition to `guard let feeAmount = transfer.destinationFeeAmount, Double(feeAmount) != 0 else {...} `